data/forms/child-referral.jsx

```diff
@@ -0,0 +1,565 @@
+export default {
+  title: 'FAST referral form',
+  path: '/form/child-referral/',
+  steps: [
+    {
+      id: 'client-details',
+      title: 'Client Details',
+      components: [
+        {
+          component: 'TextInput',
+          name: 'worker_email',
+          width: '30',
+          label: 'Email address',
+          hint: 'Valid email address',
+          rules: { required: true },
```

Comment:
We should check that is a correct email

Reply:
Added new email input component, test & storybook